### ISSUE TITLE
Table has invalid aria-readonly attribute

### ISSUE THREAD

**Issue by calesce** [bvaughn/react-virtualized#1313]
You can see this in the [current demo for Table](https://bvaughn.github.io/react-virtualized/#/components/Table) and navigating to the `ReactVirtualized__Table__Grid` in devtools.

### What is the current behavior?

The `ReactVirtualized__Table__Grid` has attribute `aria-readonly` set to `true` by default.

### What is the expected behavior?

There should not be an `aria-readonly` attribute set on an element with role: `rowgroup`. See [ARIA 1.1](https://www.w3.org/TR/wai-aria-1.1/#rowgroup). It doesn't break app behavior but in violating a standard it triggers errors in some accessibility testing tools.
